### PR TITLE
Update Developer Guide - Use cases

### PR DESCRIPTION
**Pull request by Ardentsoul**
Updated use cases UC01-UC07.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -304,12 +343,146 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    Use case ends.
+
+**Extensions**
+
+* 1a. There is further input after the user enters exit. E.g. `exit now`
+
+    * 1a1. Trackermon will only register the `exit` command.
+
+      Use case resumes at step 2.
+---
```
Comment (JonathanHoshi) [inline docs/DeveloperGuide.md:481]: Are we implementing the exit command for v1.2? Just checking to see if we might want to add that to our feature list if it seems needed.

**Review by JonathanHoshi — COMMENTED** [review]
Good job!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (aweijun) [reply docs/DeveloperGuide.md:481]: Seconded. I think that an exit command seems like a basic feature for a minimal working product. We can confirm with the rest during the next team meeting. In this case, we should also consider adding it into the user guide as while.

**Review by aweijun — APPROVED** [review]
Looks good to me! Other than the exit command portion everything else seems to be in order!

**Review by JonathanHoshi — APPROVED** [review]
LGTM!

**Pull request merged**